src/scanners/filename.js

```diff
@@ -7,6 +9,8 @@ export default class FilenameScanner extends BaseScanner {
 
   scan() {
     return new Promise((resolve) => {
+      var ext = extname(this.filename);
```

Comment:
I'd use `const` here instead of `var` (I know some of the linter code was using `var`, but might as well use `const/let` going forward).

Also I'd call it `extension`, we have the space!